### PR TITLE
Updated  jsconfig.json file for navigation and autocomplete

### PR DESCRIPTION
**Pull request by Prasannams97**
Fixes: #727

### CODE REVIEW

**Comment by Prasannams97** [comment]
@yedhink _a Please review this PR.

File: cypress-tests/cypress/jsconfig.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["./node_modules/cypress", "cypress/**/*.js"],
+  "include": ["**/*", "cypress/**/*.js"],
```
Comment (yedhink) [inline cypress-tests/cypress/jsconfig.json:2]:
```suggestion
  "include": ["./**/*", "cypress/**/*.js"],
```

**Review by yedhink — COMMENTED** [review]
@Prasannams97 _a

Have mentioned a minor update.

Please check and verify.

File: cypress-tests/cypress/jsconfig.json (hunk 1 above)
Comment (Prasannams97) [reply cypress-tests/cypress/jsconfig.json:2]: Getting no input were found in config file error, if I made the above changes.

**Comment by yedhink** [comment]
@Prasannams97 _a

>Getting no input were found in config file error, if I made the above changes.

Please attach a screenshot or a video showing the exact error and where and when it's occurring. Moving forward, if you encounter an error, then make sure to attach something solid like a screenshot showing what/where exactly this error is occurring.

Thanks.

**Comment by Prasannams97** [comment]
> @Prasannams97 _a
> 
> > Getting no input were found in config file error, if I made the above changes.
> 
> Please attach a screenshot or a video showing the exact error and where and when it's occurring. Moving forward, if you encounter an error, then make sure to attach something solid like a screenshot showing what/where exactly this error is occurring.
> 
> Thanks.

@yedhink _a
Below is the screenshot

Screenshot 2021-11-23 at 5 13 33 PM

**Comment by yedhink** [comment]
@Prasannams97 _a

Thoroughly and very carefully go through my suggestion once again and make sure you add my suggestion exactly as I have pointed out.

Currently, from the screenshot, the suggestion added is NOT the same as what I had suggested.

File: cypress-tests/cypress/jsconfig.json (hunk 1 above)
Comment (Prasannams97) [reply cypress-tests/cypress/jsconfig.json]: Added the changes.

**Comment by Prasannams97** [comment]
@yedhink _a Please check.

**Pull request merged**